docs/users/coaches.md

```diff
@@ -23,32 +23,33 @@ You can add new coaches using the _Add New Coach_ option, and edit them by click
 
-## Assigning coaches to students
+## Assigning coaches to cohorts of a course
 
-Once a coach has been added to a school, they can immediately access all communities. However, to review submissions from students, they must be assigned to students in one of two ways:
+Once a coach has been added to a school, they can immediately access all communities. However, to review submissions from students, they must be assigned to a cohort of a course.
 
 ### Course Coaches
 
 To assign a coach to a course, head to the _Coaches_ sub-menu inside a course:
 
 ![Coaches assigned to a course](../assets/coaches/course_coaches_page.png)
 
-This assignment allows coaches to review submissions from _all_ students in a course. For courses with small numbers of students, this is probably sufficient configuration.
+This page lists all coaches who have been assigned to the course. To add a new coach, click on the _Assign Coaches to Course_ button. This will open a form that allows you to select coaches from the list of all coaches in the school and select the cohorts that they should be assigned to.
 
-However, if your list of students is large, you may want to split the reviewing workload among a number of coaches. This is made easier by directly assigning course coaches to students.
+Coaches will then be able to review submissions exclusively from the students within the cohorts to which they have been assigned.
 
-### Assigning coaches to students and teams
+### Assigning coaches to students
+
+To assign a coach to a student, head to the [students](/users/students#editing-student-details) tab inside a course and click on a student to edit their details. This will open a form which has a field titled _Personal Coaches_ which allows you to select coaches from the list of all coaches in the course.
 
 Head to the list of students in a course, and select any student to edit their details. If you've picked a team, you'll notice a field titled _Team Coaches_, and if you've picked a lone student, then the same field would be labled _Personal Coaches_.
 
 This editable list of coaches allows you to directly assign coaches to a student or a team.
 
 This makes a few changes across Pupilfirst:
 
-1. The list of [submissions to review](/users/reviewing_submissions) can now be filtered by selecting an _assigned coach_.
-2. When a coach with directly assigned students loads the page for the first time, the filter defaults to show them only submissions from their own students.
-3. When viewing the details of a submission, the directly assigned coaches are listed at the top.
-4. When browsing the [coach's view of student cohorts](/users/cohort_reports), similar filters are available, and the assigned coaches are listed on both the _index_ page, and within the student report itself.
```

Comment:
I think the filter is still available 🤔

Reply:
Updated it.

+ 1. The list of [submissions to review](/users/reviewing_submissions) can now be filtered by selecting a _Personal Coach_.